components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/SendLocationMessage.java

```diff
@@ -51,14 +48,6 @@ public void setLivePeriod(Integer livePeriod) {
         this.livePeriod = livePeriod;
     }
 
-    public boolean isDisableNotification() {
-        return disableNotification;
-    }
-
-    public void setDisableNotification(boolean disableNotification) {
-        this.disableNotification = disableNotification;
-    }
-
```

Comment:
Is there a reason to remove this field?

Reply:
That field was added by mistake recently.
That fiels is inherited from: https://github.com/apache/camel/blob/0e0aa95517d117ba6f12504a40dc61492f89e988/components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/SendLocationMessage.java#L21

https://github.com/apache/camel/blob/d0e04616cebee03ecdd5ed5333ed57a93237b62c/components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/OutgoingMessage.java#L34